src/manipulation.js

```diff
@@ -159,6 +159,10 @@ function fixInput( src, dest ) {
 }
 
 jQuery.extend({
+	htmlPrefilter: function( html ) {
```

Comment:
Anyone overriding this method should keep and use a reference to what they're replacing, as in the test cases added here.